android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/MainActivity.java

```diff
@@ -1,25 +1,142 @@
 package com.sdl.hellosdlandroid;
 
 import android.content.Intent;
+import android.content.SharedPreferences;
 import android.os.Bundle;
+import android.preference.PreferenceManager;
 import android.support.v7.app.AppCompatActivity;
+import android.text.Editable;
+import android.text.TextWatcher;
 import android.view.Menu;
 import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import java.util.Timer;
+import java.util.TimerTask;
 
 public class MainActivity extends AppCompatActivity {
 	private static final String TAG = "MainActivity";
-	
+
+	public static final String IP = "ip";
+	public static final String PORT = "port";
+	public static final String COMMAND = "command";
```

Comment:
@santhanamk Ip and port are used to pass intent, thought, still this change is related more to demonstrate proposal implementation, rather than to a functional one. If it is required, I can separate this code, but, I believe, this should not go to the final release version and should be deleted. Though, it remains here right now in order to help everybody to acknowledge. Should I remove it now?